frontend/src/main/java/de/iav/frontend/controller/CreateOrEditMeetingController.java

```diff
@@ -0,0 +1,227 @@
+                } else {
+                    Alerts.getMessageBoxWithWarningAndOkButton(
+                            CREATE_NOT_POSSIBLE,
+                            "Anlegen eines Meetings nicht möglich weil bereits ein existiert mit ID: " + meetingExist.id(),
+                            "kommt später");
+                }
+            }
+        } catch (IOException e) {
+            Alerts.getMessageBoxWithWarningAndOkButton(CREATE_NOT_POSSIBLE, "Hat leider nicht geklappt. IOException Siehe Meldung:", e.getMessage());
+        } catch (InterruptedException e) {
+            Alerts.getMessageBoxWithWarningAndOkButton(CREATE_NOT_POSSIBLE, "Hat leider nicht geklappt. InterruptedException Siehe Meldung:", e.getMessage());
+            throw e;
+        }
+    }
+
+    private void updateOneMeeting(ActionEvent event) throws InterruptedException {
+        try {
+            if (checkDate() && checkLocation()) {
+                Meeting meetingExist = meetingService.getMeetingByDateTimeAndLocation(meetingToUpdate.dateTime(), meetingToUpdate.location());
+                if (meetingExist != null) {
+
+                    Meeting meeting = new Meeting(meetingToUpdate.id(), lDateTime.getText(), cbLocation.getValue(), meetingToUpdate.speechContributionList());
+                    Meeting savedMeeting = meetingService.updateMeeting(meeting, authService.getSessionId());
+                    if(Alerts.getMessageBoxWithInformationAndOkButtonAndReturnBoolean("Erfolgreiches Aktualisieren des Meetings mit der ID",
+                            "Erfolgreiches Aktualisieren des Meetings mit der ID: " + savedMeeting.id(), "alles geklappt"))
+                        sceneSwitchService.switchToMeetingController(event, loggedUser, meetingIndex);
+                } else {
+                    Alerts.getMessageBoxWithWarningAndOkButton(
+                            UPDATE_NOT_POSSIBLE,
+                            "Aktualisieren eines Meetings nicht möglich weil ein Meeting nicht existiert!",
+                            "kommt später");
+                }
+            }
+        } catch (IOException e) {
+            Alerts.getMessageBoxWithWarningAndOkButton(CREATE_NOT_POSSIBLE, "Hat leider nicht geklappt. IOException Siehe Meldung:", e.getMessage());
+        } catch (InterruptedException e) {
+            Alerts.getMessageBoxWithWarningAndOkButton(CREATE_NOT_POSSIBLE, "Hat leider nicht geklappt. InterruptedException Siehe Meldung:", e.getMessage());
+            throw e;
+        }
+    }
+
+    private List<SpeechContribution> getSpeechContributionList(List<TimeSlot> timeSlots)
+    {
+        List<SpeechContribution> speechContribution = new ArrayList<>();
+        for (TimeSlot timeSlot:timeSlots) {
+            speechContribution.add(new SpeechContribution(null, timeSlot, null, null));
+        }
+        return speechContribution;
+    }
+
+    public void setUserAndEditToShow(User user) {
+        loggedUser= user;
+        meetingToUpdate = null;
```

Comment:
Absichtlich null setzen kann ganz schnell in die Hose und zu Fehlern führen - muss das hier so sein?

Reply:
wir können uns gerne über alternative Lösung unterhalten unterhalten. Im fall eines neues Meetings, ist das meetingToUpdate nicht nötig und wird nicht angefragt.